test/base_test_fixture.h

```diff
@@ -1502,6 +1502,38 @@ struct base_test_fixture
         results.get_ref(0, ref);
         REQUIRE(ref == name);
     }
+    void string_vector_test()
```

Comment:
Yes you are right, I did not understand how these tests were being incorporated into each driver test suite when I wrote this, will fix.